doc/user/command-line.xml

```diff
@@ -1003,18 +998,19 @@ bar.c
 
       <para>
 
-      NOTE:  Before &SCons; release 0.98.1, these build variables
-      were known as "command-line build options."
-      The class was actually named the &Options; class,
-      and in the sections below,
-      the various functions were named
+      Historical note:  In old &SCons; (prior to 1.0),
```

Comment:
I'd leave the version 0.98.1. There were a handful of versions between that and 1.0.0 (which hopefully aren't still being used) so better to be accurate about this.

Reply:
Ok.

Reply:
Changed.